src/test/java/no/sikt/nva/scrapers/ExcelScraperTest.java

```diff
@@ -0,0 +1,96 @@
+package no.sikt.nva.scrapers;
+
+import static no.sikt.nva.validators.ExcelRowValidator.ERROR_MESSAGE_INVALID_EMBARGO_FORMAT;
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import java.nio.file.NoSuchFileException;
+import no.sikt.nva.exceptions.ExcelException;
+import org.junit.jupiter.api.Test;
+
+class ExcelScraperTest {
+
+    private final String VALID_EXCEL_FILE = "src/test/resources/excelScraper/Metadata.xlsx";
+    private final String INVALID_EXCEL_FILE = "src/test/resources/excelScraper/MetadataInvalid.xlsx";
+    private final String EMPTY_EXCEL_FILE = "src/test/resources/excelScraper/EmptyFile.xlsx";
+    private final String INVALID_EXCEL_FILE_EMBARGO_DATE = "src/test/resources/excelScraper"
+                                                           + "/MetadataInvalidEmbargoDate.xlsx";
+
+    @Test
+    void shouldNotThrowExceptionWhenValidExcelFileIsProvided() {
```

Comment:
I think there is missing a couple of  "and"'s here from a TDD stand point. It will not throw exception if the validator says it's okay. It converted it successfully to records, and pushed the result to aws.

Reply:
Converting to records, and subsequently pushing to aws, are next on the task list 👍